source/includes/api/_metadata.md

```diff
@@ -1,188 +1,436 @@
-</aside>
+For example, metadata about a tree cover dataset in [Global Forest Watch](https://www.globalforestwatch.org/) might be available in both English and Spanish (each a metadata different object), and these may be different compared to metadata objects in the same languages associated with [Resource Watch](https://resourcewatch.org/).
 
-> Create metadata for a dataset
+***Note - while it maybe tempting to skip the creation of detailed metadata objects for a dataset, layer or widget, to help with discoverability and usability, it is highly recommended to invest time needed to create quality metadata.***
+
+> Overview of a metadata object
+
+| Field         | Type   | Required | Description |
+| ------------- | ------ | -------- | ----------- |
+| `application` | String | Yes      | RW API application associated with this object. |
+| `language`    | String | Yes      | Language of this objects values, e.g., `"en"`, `"es"`. |
+| `dataset`     | String | Yes      | Identifier of the Dataset associated with this object. |
+| `resource`    | Object | No       | Object describing the identifier (`id`) and type (`type`) of the Layer or Widget associated with this object. |
+| `name`        | String | No       | Name associated with this object. |
+| `description` | String | No       | Description associated with this object. |
+| `source`      | String | No       | Indication of the original source of this metadata. |
+| `citation`    | String | No       | Citation associated with this object. |
+| `license`     | String | No       | Reuse licensing terms associated with this object. |
+| `info`        | Object | No       | Object containing any number of undefined fields. |
+| `units`       | String | No       | Measurement units of the data variable associated with this object. |
+
+## Overview of available endpoints
+
+Metadata objects are always associated with a Dataset, and may also be associated with Layer or Widget. Hence, the endpoints for interacting with metadata objects are always sub-paths of `/dataset` and their associated `/layer` or `/widget`.
+
+Any user can retrieve metadata objects, however to create, update or delete and object the appropriate [authentication](#authentication) credentials are required; a `403` `Forbidden` error is returned if authorization fails.
+
+Credentials required for creating, updating or deleting:
+
+- You must have an `ADMIN` or `MANAGER` role.
+- You must belong to the same `application` group as the dataset associated with the metadata object.
+
+> Overview of endpoints
+
+| Method | Path                                | Description                                                                                                                       |
+|--------|-------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
We are missing the clone endpoint here: [`router.post('/dataset/:dataset/metadata/clone', cloneValidationMiddleware, authorizationMiddleware, MetadataRouter.clone);`](https://github.com/resource-watch/rw_metadata/blob/36bada45c4d8e567d9c890f8c9de6edf9276771c/app/src/routes/api/v1/metadata.router.js#L242)

Reply:
Having some difficulty working out how to use `clone`; not sure on the correct body to POST